README.md

```diff
@@ -302,6 +301,11 @@ If you run Debian Buster (currently Debian testing) or Debian sid, ripgrep is
 $ sudo apt-get install ripgrep
 ```
 
+From Cosmic (18.10), ripgrep is also [available in Ubuntu](https://launchpad.net/ubuntu/+source/rust-ripgrep) (same packaging as Debian).
+```
+$ sudo apt-get install ripgrep
```

Comment:
Oh interesting. What is the difference between the `ripgrep` and `rust-ripgrep` packages?

Reply:
The source package is called `rust-ripgrep`, the binary package is called `ripgrep`